addons/website_payment/__manifest__.py

```diff
@@ -12,6 +12,7 @@
         'website',
         'payment',
         'portal',
+        'account_payment',
```

Comment:
could replace the payment dependency (but not really important)

Reply:
done in https://github.com/odoo/odoo/pull/90899/commits